src/lang/en.json

```diff
@@ -2247,6 +2247,11 @@
   "faction.tags.warlike": "Warlike",
   "faction.tags.warlike.description": "There are factions with a military orientation, and then there are factions that just really love killing things. Whether or not this faction has developed sophisticated military resources and techniques, the membership is zealous of battle and has a positive taste for fighting.",
   "faction.tags.warlike.effect": "Once per turn, this faction can roll an additional d10 when making a Force attack.",
+  "entity.settings": "Settings",
+  "settings.hexSystemText": "Hex System Text",
+  "settings.showEntityCount": "Show Entitiy Count",
```

Comment:
Misspelling here. `Show Entity Count`.

Reply:
Fixed in Irame/sectors-without-number@409bf0e47943ef830eeb4e80ac897330ef7e18cb